src/olympia/constants/base.py

```diff
@@ -212,16 +212,16 @@
     'utm_source': 'addons.mozilla.org',
 }
 VALID_CONTRIBUTION_DOMAINS = (
-    'buymeacoffee.com',
+    'www.buymeacoffee.com',
     'donate.mozilla.org',
     'flattr.com',
     'github.com',
     'ko-fi.com',
     'liberapay.com',
-    'micropayment.de',
+    'www.micropayment.de',
     'opencollective.com',
-    'patreon.com',
-    'paypal.com',
+    'www.patreon.com',
+    'www.paypal.com',
     'paypal.me',
```

Comment:
to work out which was the canonical domain I just loaded up each site without the `www.` and saw if it redirected.